pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -43,7 +43,8 @@
     SamplerV2,
     RuntimeJob,
 )
-from qiskit.providers.models import BackendProperties, QasmBackendConfiguration  # type: ignore
+from qiskit_ibm_runtime.models.backend_configuration import PulseBackendConfiguration  # type: ignore
```

Comment:
Why replacing `QasmBackendConfiguration` with `PulseBackendConfiguration`?

Reply:
Since we switched to using qiskit runtime, this is actually the type of what `IBMBackend.configuration()` returns. This change only corrects the typing annotations.